source/portable/NetworkInterface/SH2A/CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ target_sources( freertos_plus_tcp_network_if
 
 # target_link_libraries( freertos_plus_tcp_network_if
 #   PRIVATE
-#     hwEthernetLib # Todo where is hwEthernet.h?
```

Comment:
What about this?

Reply:
Actually I can't find this file in this repo.
But to include hwEthernet.h in NetworkInterface.c, that file must be included either by manually added to the include path or link to another library. So I'd suggest to keep it here.